README.md

```diff
@@ -167,82 +170,76 @@ What we see here are (the icons are somewhat arbitrary as we have no finished UI
 6. We will go though Phase 5 and Phase 6, until we reach the ConditionalGateway which will check if we need another iteration of the loop and if so, loop back to Phase 5, otherwise it will send a signal to the flow to the right
 7. The END event dispatcher is activated and dispatches the END event, marking the end of the process.
 
-The data schema for such a process would look like this (handwritten, take with a grain of salt):
+The data schema for such a process would look like this (handwritten, take with a grain of salt). The element IDs will also be uuids (for simplification I'm using shorter IDs)!
 
 ```js
 const workflow = {
   id: 'a6921701-4706-40e9-82a0-07aa7e86987b',
   version: '1',
-  elements: {
-
-    eventListeners: [
-      {
-        type: 'START',
-        id: 'eventListener_1',
-        elementType: 'EVENT_LISTENER',
-      },
-      {
-        type: 'TIMER',
-        id: 'eventListener_2',
-        durationInMS: 100000,
-        phaseId: 'phase_1',
-        elementType: 'EVENT_LISTENER',
-      },
-      {
-        type: 'APPROVAL',
-        id: 'eventListener_3',
-        phaseId: 'phase_2',
-        elementType: 'EVENT_LISTENER',
-      },
-      {
-        type: 'CONDITION',
-        id: 'eventListener_4',
-        phaseId: 'phase_2',
-        condition: {
-          left: {
-            type: 'SENSOR_DATA',
-            data: {
-              reporterFctId: 'b505cf08-cc3f-4814-8d76-3dd23ad1ea35',
-            },
+  elements: [
+    {
+      type: 'START',
+      id: 'eventListener_1',
+      elementType: 'EVENT_LISTENER',
```

Comment:
As I mentioned, we have only a single `elements` array now, and the elements carry their element time. This made the logic more convenient because often we pass around elements and we often need to know what element type they actually are.